deployment/ccip/shared/stateview/aptos/state.go

```diff
@@ -20,9 +25,11 @@ import (
 )
 
 type CCIPChainState struct {
-	MCMSAddress      aptos.AccountAddress
-	CCIPAddress      aptos.AccountAddress
+	MCMSAddress aptos.AccountAddress
+	CCIPAddress aptos.AccountAddress
+
 	LinkTokenAddress aptos.AccountAddress
+	ManagedTokens    map[shared.TokenSymbol]aptos.AccountAddress
```

Comment:
This is never being set, is it?

Reply:
This could be set on `loadState`. Tokens will have a label with their symbol in AddressBook. eg:
```    
"0xc3522eda74096ab89bdd7ae1533a34fe1f2a3ec9e21e804d006ac53a8ca1365f": {
      "Type": "AptosManagedTokenType",
      "Version": "1.6.0",
      "Labels": {
        "CCIP-BnM": {}
      }
    },
```